vipps-partner-api.md

```diff
@@ -17,26 +20,29 @@ Document version 1.5.6.
 
 ## Table of contents
 
-* [Information for Vipps partners](#information-for-vipps-partners)
-  * [Integrating with this API](#integrating-with-this-api)
-  * [Partner keys](#partner-keys)
-* [Get information about a merchant based on organization number](#get-information-about-a-merchant-based-on-organization-number)
-  * [Future improvements](#future-improvements)
-  * [In the meantime](#in-the-meantime)
-* [Get information about a sale unit based on MSN](#get-information-about-a-sale-unit-based-on-msn)
-  * [Future improvements](#future-improvements)
-  * [In the meantime](#in-the-meantime)
-* [Submit a product order for a merchant](#submit-a-product-order-for-a-merchant)
-  * [Future improvements](#future-improvements)
-* [Future plans for this API](#future-plans-for-this-api)
-* [Questions?](#questions)
+- [Vipps Partner API](#vipps-partner-api)
```

Comment:
The title and ToC heading should not be in the ToC. 🙈 The rest of the headings should then move one step top the left.